spec/directives/import/error/import_relative.hrx

```diff
@@ -0,0 +1,27 @@
+<===> input.scss
+// Importing a file must not allow that file to resolve its URL
+// relatively to the importer but only to itself (and to load paths)
```

Comment:
Also, can you add a note that this is a regression test and a reference to the issue in scssphp that has details about the bug?